src/test/kotlin/org/opensearch/replication/integ/rest/StartReplicationIT.kt

```diff
@@ -1147,6 +1147,12 @@ class StartReplicationIT: MultiClusterRestTestCase() {
     }
 
     fun `test that wait_for_active_shards setting is updated on follower through start replication api`() {
+
+        //Ignore this test if clusters dont have multiple nodes
+        if(!isMultiNodeClusterConfiguration(LEADER, FOLLOWER)){
+            return
+        }
+
```

Comment:
This pattern is super dangerous as we may assume tests succeeded and validated the codepaths, but in reality got skipped.

Can we figure out a mechanism to "skip" the test execution itself instead of short circuting?

Reply:
I think we can use `LuceneTestCase.assumeTrue`. Our ITs already extend this class.

Reply:
Updated this in next revision.